nansat/tests/mappers/test_mapper_opendap_mywave.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from nansat.mappers.mapper_opendap_mywave import Mapper
+from netCDF4 import Dataset
+
+
+class MyWaveOpenDAPTests(unittest.TestCase):
+
+    def setUp(self):
+        self.src = 'http://thredds.met.no/thredds/dodsC/fou-hi/mywavewam4archive' \
```

Comment:
This test package should not use real data but rather mocks. The goal is to add mapper testing to the continuous integration, and then accessing real data is too time consuming. Could you rather rewrite the tests using mocking, or move it to the integration tests (which are meant to be run locally)?

Reply:
I test a parsing of the src (as a str) and do not exploit (I/O) the data from the link